core/controllers/admin_test.py

```diff
@@ -63,8 +63,14 @@
 BOTH_MODERATOR_AND_ADMIN_EMAIL = 'moderator.and.admin@example.com'
 BOTH_MODERATOR_AND_ADMIN_USERNAME = 'moderatorandadm1n'
 
-PARAM_NAMES = python_utils.create_enum('test_feature_1')  # pylint: disable=invalid-name
-FEATURE_STAGES = platform_parameter_domain.FEATURE_STAGES
+
+class ParamNames(enum.Enum):
+    """Enum for parameter names."""
+
+    TEST_FEATURE_1 = 'test_feature_1'
+
+
+FEATURE_STAGES = platform_parameter_domain.FEATURE_STAGES # pylint: disable=invalid-name
```

Comment:
Do we still need the bypass flag here? `# pylint: disable=invalid-name`

If yes, can we use `platform_parameter_domain.FEATURE_STAGE` directly in the test and remove this line?

Reply:
Done